src/WebJobs.Script/Description/Rpc/ScriptInvocationResult.cs

```diff
@@ -5,7 +5,7 @@
 
 namespace Microsoft.Azure.WebJobs.Script.Description
 {
-    internal class ScriptInvocationResult
+    public class ScriptInvocationResult
```

Comment:
Same question about the accessibility change here.

Reply:
Switched it back to internal. Here is the reason for rest of the types
Added IFunctionDispatcher as a dependency to ScriptHost --> This required making IFunctionDispatcher public --> This required making FunctionRegistrationContext public